addons/pos_cache/models/pos_cache.py

```diff
@@ -62,34 +56,35 @@ def _get_oldest_cache_time(self):
             oldest_cache = pos_cache.search([('config_id', '=', cache.id)], order='write_date', limit=1)
             cache.oldest_cache_time = oldest_cache.write_date
 
-    # Use a related model to avoid the load of the cache when the pos load his config
     cache_ids = fields.One2many('pos.cache', 'config_id')
```

Comment:
This comment means: we created new model ``pos.cache``, because we don't  want to add new fields direct to pos.config as it leads to loading them on loading pos.config record.

I.e. it was basically one2one field.
With the new commit, it's not more the case.